pgh_play/www/php/interface.php

```diff
@@ -201,5 +203,72 @@ function get_parks(){
 	return $results;
 }
 
+// enter new park into the db via new park form
+// this will store the following info in the db: Park name, address, rating, ages, comments
+//  all of these inputs are received by the new park form a user fills out
+// *****facilities and pictures still need to be implemented 
+function new_park(){
```

Comment:
Something like this might do the trick:
{ "name":"park name",
 "address":"street addr",
 "city":"city name"
 "gps_long":234.234234,
 "gps_lat":234.234234,
 "rating":4.2,
 "infant_safe":true,
 "toddler_safe":true,
 "five_eight":true,
 "nine_twelve":true,
 attributes: [  1, 2, 3, 4, 5, 6, 7 ]
}

In javascript,  var output;  output['name'] = "Green Tree Park";  ...  output['attributes'] = new Array();  output['attributes'][0] = 1; ...  (arrays need to start with 0 for proper json encoding)

In php,  $obj = json_decode($_POST['obj'], true);
$obj will have same structure as the var output did.